src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5145,6 +5145,12 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_TupleInferredNamesNotAvailable" xml:space="preserve">
     <value>Tuple element name '{0}' is inferred. Please use language version {1} or greater to access an element by its inferred name.</value>
   </data>
+  <data name="WRN_AttributesOnBackingFieldsNotAvailable" xml:space="preserve">
```

Comment:
>WRN_AttributesOnBackingFieldsNotAvailable [](start = 14, length = 41)

I am not sure why is this a warning. It looks like other similar diagnostics are errors.

Reply:
Because such code produced a warning with C# 7.2 (and older) compilers.
 `warning CS0657: 'field' is not a valid attribute location for this declaration. Valid attribute locations for this declaration are 'property'. All attributes in this block will be ignored.`